mapit/static/mapit/sass/modules/area_list.scss

```diff
@@ -1,46 +0,0 @@
-// Number of Levels
-$area_level: 10;
-
-// Amount to indent
-$indentation: 2.5%; // ~ 10px
-
-// Full width of list
-$list_width: 33.75em;
-
-.area_list {
-
-    @include resetlist;
-    display: block;
-    width: $list_width;
-    max-width: 100%;
-
-    li {
-        display: block;
-        background-color: #e5e5e5;
-        padding: 0.625em;
-        @include border-radius(0.625em);
-        margin-bottom: 0.625em;
-
-        h3 {
-            margin: 0;
-            font-size: 1.125em;
-        }
-        p {
-            margin: 0;
-            font-size: 0.75em;
-        }
-    }
-}
-
-// Creates area_level_1 to area_level_10
-// optionally add to list items to add indentation suggesting hierarchy
-
-@while $area_level > 0 {
```

Comment:
It would be a shame to lose this (which indents the area lists on e.g. http://global.mapit.mysociety.org/point/4326/8.55,47.366667.html).